lib/fhir_models/bootstrap/definitions.rb

```diff
@@ -216,21 +216,27 @@ def self.get_codes(uri)
     def self.get_display(uri, code)
       return nil if uri.nil? || code.nil?
       load_expansions
-      valuesets = @@expansions.select { |x| x['compose']['include'].detect {|i| i['system'] == uri } }
-      valuesets += @@valuesets.select { |x| x['url'] == uri }
+      valuesets = @@expansions.select { |ex| ex['compose']['include'].detect { |i| i['system'] == uri } }
+      valuesets += @@valuesets.select { |vs| vs['url'] == uri }
       code_hash = nil
       valuesets.each do |valueset|
-        if !valueset['expansion'].nil? && !valueset['expansion']['contains'].nil?
-          code_hash = valueset['expansion']['contains'].select { |x| x['system'] == uri && x['code'] == code}.first
-        elsif !valueset['compose'].nil? && !valueset['compose']['include'].nil?
-          included_systems = valueset['compose']['include'].map { |x| x['system'] }.uniq
-          systems = @@valuesets.select { |x| x['resourceType'] == 'CodeSystem' && included_systems.include?(x['url']) }
-          systems.each do |x|
-            code_hash = x['concept'].select { |con| con['url'] == uri && con['code'] == code }.first if x['concept']
+        if valueset['expansion'] && valueset['expansion']['contains']
+          # This currently only matches 'expansions', not 'valuesets'
```

Comment:
I was only able to find 2 `valuesets` with `ValueSet.expansions.contains`, and their `system` value never equaled the `ValueSet.url` (line 220, shown above).

Would this ever be a valid logic path for `valuesets` or is it only realistic for `expansions`?